Source/WebCore/css/CSSCounterStyle.cpp

```diff
@@ -313,5 +313,7 @@ void CSSCounterStyle::extendAndResolve(const CSSCounterStyle& extendedCounterSty
         setAdditiveSymbols(extendedCounterStyle.additiveSymbols());
     if (!explicitlySetDescriptors().contains(CSSCounterStyleDescriptors::ExplicitlySetDescriptors::SpeakAs))
         setSpeakAs(extendedCounterStyle.speakAs());
+
+    setFirstSymbolValueForFixedSystem(extendedCounterStyle.firstSymbolValueForFixedSystem());
```

Comment:
Can we move this next to the `setSystem(extendedCounterStyle.system());` call above in the function? They go together.